### PR TITLE
kubernetes secrets "\n" problem FIX.

### PR DESCRIPTION
**Pull request by DakshMiglani**
Fixes: #732

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-nodejs?pullRequest=773) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-nodejs?pullRequest=773) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-nodejs?pullRequest=773) it.</sub>

File: TROUBLESHOOTING.md [hunk 1]
```diff
@@ -86,6 +86,12 @@ becomes
 
 In the first case SENDGRID_API_KEY is in reference to the name of the environment variable, while the second case references the actual SendGrid API Key.
 
+If you're using Kubernetes Secrets and passing the API Keys to the Environment using it, You may find that there is a `\n` charachter in the environment variable. You can use the trim function to remove it like this:
```
Comment (reedsa) [inline TROUBLESHOOTING.md]: Misspelling of `character`. Otherwise looks good 👍

Comment (DakshMiglani) [reply TROUBLESHOOTING.md]: sure I'll edit.

**Comment by DakshMiglani** [comment]
Who's merging?

**Comment by thinkingserious** [comment]
Hello @DakshMiglani,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/hacktoberfest-2018-has-arrived/">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/hacktoberfest/join">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by DakshMiglani** [comment]
@thinkingserious Sure, thanks.

**Comment by thinkingserious** [comment]
Hello @DakshMiglani,<br /><br/>Thanks again for the PR!<br /><br />                                         We noticed that you have earned 1 out of the 5 points needed to receive glorious SendGrid Hacktoberfest swag.<br /><br />                                         Please take a moment to checkout <a href="https://dx.sendgrid.com/hacktoberfest/issues">this link</a> to find more issues to get you past the required threshold.<br /><br />                                         Also, please be sure that you have officially registered <a href="https://dx.sendgrid.com/hacktoberfest/join">here</a>.<br /><br />                                         Thank you and Happy Hacktobering!<br /><br />                                         Team SendGrid DX

**Comment by thinkingserious** [comment]
Hello @DakshMiglani,<br /><br/>Thanks again for the PR!<br /><br />                                         You have earned 1 out of the 5 points needed to receive glorious SendGrid Hacktoberfest swag.<br /><br />                                         Please take a moment to checkout <a href="https://dx.sendgrid.com/hacktoberfest/issues">this link</a> to find more issues to get you past the required threshold or to simply continue the celebration.<br /><br />                                         Also, <strong>please be sure you have officially registered with us <a href="https://dx.sendgrid.com/hacktoberfest/join">here</a> by November 1, 2018</strong> to qualify.<br /><br />                                         If you have any questions you can email us at <a href="mailto:dx+hacktoberfest2018@sendgrid.com">dx+hacktoberfest2018@sendgrid.com</a>.<br /><br />                                         Thank you and Happy Hacktobering!<br /><br />                                         Team SendGrid

**Pull request merged**